src/v2/components/MessageList/MessageList.tsx

```diff
@@ -135,7 +135,6 @@ type MessageListPropsWithContext<
     | 'channel'
     | 'disabled'
     | 'EmptyStateIndicator'
-    | 'initialScrollToFirstUnreadMessage'
```

Comment:
you should remove this from all the places it's being passed in too because it is now an unused value

Reply:
Its still used in Channel.tsx to disable this feature, just not in MessageList component.

Reply:
but it can be removed from the contexts then if it's only used in Channel

Reply:
Ahh ok. Makes sense

Reply:
@virdesai Done